docs/sources/flow/reference/cli/convert.md

```diff
@@ -2,24 +2,27 @@
 canonical: https://grafana.com/docs/agent/latest/flow/reference/cli/convert/
 labels:
   stage: beta
-title: grafana-agent convert
+title: grafana-agent-flow convert
 weight: 100
 ---
 
-# `grafana-agent convert` command
+# `grafana-agent-flow convert` command
 
-The `grafana-agent convert` command converts a supported configuration format
+The `grafana-agent-flow convert` command converts a supported configuration format
 to Grafana Agent Flow River format.
 
 ## Usage
 
-Usage: `grafana-agent convert [FLAG ...] FILE_NAME`
+Usage: 
+
+* `grafana-agent-flow convert [FLAG ...] FILE_NAME`
+* `AGENT_MODE=flow grafana-agent convert [FLAG ...] FILE_NAME`
```

Comment:
I'd personally swap the order of these two; it's more common to have a `grafana-agent` binary where you set `AGENT_MODE=flow` than it is to have the `grafana-agent-flow` binary. 

The `grafana-agent` binary is always used on Docker and Kubernetes deployments, which are our most common type of deployment.

Reply:
That works for me.

Reply:
Fixed in a new commit.